fastlane_bot/tests/test_037_Exchanges.py

```diff
@@ -174,7 +173,7 @@ def test_test_uniswap_v2_exchange_fork():
     async def test_uniswap_v2_exchange_fork():
```

Comment:
Unrelated to this PR but I need this comment to make it another issue

```
async def test_uniswap_v2_exchange_fork():
        assert (pancake_v2_exchange.exchange_name in "pancakeswap_v2"), f"Wrong exchange name. Expected pancakeswap_v2, got {pancake_v2_exchange.exchange_name}"
        assert (pancake_v2_exchange.base_exchange_name in "uniswap_v2"), f"Wrong base exchange name. Expected uniswap_v2, got {pancake_v2_exchange.base_exchange_name}"  
```

The whole point of defining constants is that they abstract from the underlying values. The only reason why we define

`PANCAKESWAP_V2_NAME = "pancakeswap_v2"`

is because this makes debugging easier. However, if at one point we would like to change 

`PANCAKESWAP_V2_NAME = 1234`

then the code should still work as previously.

Relatedly -- tests are also contracts with the users of the library. If you do 

`assert PANCAKESWAP_V2_NAME == "pancakeswap_v2"`

then this means that the user of the library can rely on this association being true (and in SemVer this means that if ever you can it you need to change the major version number)